app/javascript/App/components/NewsfeedPage/SearchCoins.js

```diff
@@ -0,0 +1,78 @@
+import React, { Component } from 'react'
+import coinSearchProvider from '../../containers/coinSearch'
+import Input from '../Input'
+import Icon from '../Icon'
+import { union } from 'lodash'
+
+class SearchCoins extends Component {
+  selectedCoins = () => {
+    const { activeFilters } = this.props
+    const filter = activeFilters.find((filter) => filter.get('key') === 'coins')
+    if (!filter) return []
+    return filter.get('value').toJS()
+  }
+  handleSearchInput = (value) => {
+    let { searchCoins } = this.props
+    const name_not_in = this.selectedCoins()
```

Comment:
the convention in js and in our app is mostly camel case. was there a reason you chose to use snake case here?

Reply:
Yes in JS I always cameCase, however in Ruby it's snake case, and in this instance it's easier to pass in the param that the controller needs. Otherwise if the param name changes, it needs to be changed in 2 places instead of 1